client/src/addirsync.rs

```diff
@@ -1,6 +1,7 @@
 use crate::LdapClient;
 use crate::*;
 use base64urlsafedata::Base64UrlSafeData;
+use ldap3_proto::control::LdapControl;
```

Comment:
This depends on the control move PR?

Reply:
Looks like it does.

> this change is based on https://github.com/kanidm/ldap3/pull/42, which should be merged first.

@irvingoujAtDevolution I suggest to mark the PR as draft to avoid confusion. Ideally, the diff should be generated against the base, but in this case I believe it would not be possible because the base is in our fork.